### PR TITLE
Fix bug in export to kaldi function

### PR DESCRIPTION
**Pull request by HuangZiliAndy**
Hi @desh2608 @jtrmal @pzelasko , I meet some problems when I am converting a meeting directory to Kaldi. The function is giving an incorrect number of segments. Thanks!

### CODE REVIEW

**Comment by pzelasko** [comment]
Why are you losing segments if you convert recording + supervisions to cuts? It would indicate that those segments either belong to non-existent recordings or are past the recording's end time..

**Comment by HuangZiliAndy** [comment]
Hi Piotr, I don't think so. I think some are overlapping segments. I tried with another dataset and lost almost half of the segments. Let me try to find an example.

**Comment by HuangZiliAndy** [comment]
I updated in the doc above. Thanks

**Comment by pzelasko** [comment]
Ah, of course, now I see what is the issue. In segments like the following:

```
|-------seg1--------|
       |---seg2---|
```

`seg2` would be never written. Possibly the same thing happens with partially overlapping segments.

 I think your changes make sense now. Can you resolve the conflicts?

File: lhotse/kaldi.py [hunk 1]
```diff
@@ -206,11 +206,11 @@ def export_to_kaldi(
...
-    # Create a simple CutSet that ties together
-    # the recording <-> supervision information.
-    cuts = CutSet.from_manifests(
-        recordings=recordings, supervisions=supervisions
-    ).trim_to_supervisions()
+    ## Create a simple CutSet that ties together
+    ## the recording <-> supervision information.
+    #cuts = CutSet.from_manifests(
+    #    recordings=recordings, supervisions=supervisions
+    #).trim_to_supervisions()
```
Comment (pzelasko) [inline lhotse/kaldi.py:213]: Can you remove the commented code?

**Comment by pzelasko** [comment]
Once you resolve the conflicts with master, we can merge it.

File: lhotse/kaldi.py [hunk 2]
```diff
@@ -231,8 +225,8 @@ def export_to_kaldi(
         # segments
         save_kaldi_text_mapping(
             data={
-                cut.supervisions[0].id: f"{cut.recording_id} {cut.start} {cut.end}"
-                for cut in cuts
+                sup.id: f"{sup.recording_id} {sup.start} {sup.start + sup.duration}"
```
Comment (desh2608) [inline lhotse/kaldi.py:228]: You can also just use `sup.end` (`SupervisionSegment` has an `end()` [property](https://github.com/lhotse-speech/lhotse/blob/b41e4f8c57954b94790f2d05de310a11071512bc/lhotse/supervision.py#L187).

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**